### ISSUE TITLE
Why does adding a site to sidebar show the extension icon?

### ISSUE THREAD

**Issue by AzureRT456** [focus-browser/browser#250]
Added two sites (not extensions from the Firefox addons store) onto the sidebar. They don't appear with their icon like in other browsers.

**Comment by PressJump** [comment]
> Added two sites (not extensions from the Firefox addons store) onto the sidebar. They don't appear with their icon like in other browsers.

Hey @AzureRT456 
Can you send a screenshot of the sidebar for a reference? One issue may be because of a badly parsed "filtered" https prefix. Try removing the https:// prefix and if that doesn't work that means the favicon service we are using is not returning the favicon of the website. We can check if that is the case later on though.

**Comment by AzureRT456** [comment]
These are the two websites I added.

![image](https://github.com/pulse-browser/browser/assets/128042952/d1a40477-4c98-40b7-8607-0f0e50a7c52e)

I was able to fix one of the websites, however the other (https://www.lexilogos.com/keyboard/byelorussian.htm) wouldn't fix the icon. No dice when I removed the https:// or www.

**Comment by abominabledrh** [comment]
Yep, same. https://mastodon.ie/home won't show a favicon, even with the https:// removed.

**Comment by EmirYLMZ128** [comment]
![resim](https://github.com/pulse-browser/browser/assets/90498438/afbdfb22-47d1-45be-9160-9aa0e46e1a5c)
Same, whatsapp web wont show favicon.
